spikeinterface/sorters/runsorter.py

```diff
@@ -433,21 +433,27 @@ def run_sorter_container(
             volumes[str(recording_folder)] = {
                 'bind': str(recording_folder_unix), 'mode': 'ro'}
     volumes[str(parent_folder)] = {'bind': str(parent_folder_unix), 'mode': 'rw'}
-    si_dev_path = os.getenv('SPIKEINTERFACE_DEV_PATH')
-
+    
+    running_on_github_actions = os.getenv("CI") 
+    si_dev_path = None
+    if running_on_github_actions:
+        print(si_version)
+        si_dev_path = os.getenv('SPIKEINTERFACE_DEV_PATH')
+        print(si_dev_path)
+        assert si_dev_path is not None
```

Comment:
What is this ? This looks like debuging stuff.
This GH specific stuff should not be the code I think but in the test files only no ?

Reply:
Oups! you are right! I thought this was on the test side. Will push a fix!

Reply:
That was my mistake, I added when I was debugging and then did not removed then.

Reply:
No worries guys.
I just feel that you are sometimes a bit too imptatient for merges :)
Maybe the the priviliedge of youg people.